db/migrate/20230221221546_create_posts.rb

```diff
@@ -4,12 +4,12 @@ def change
       t.integer :author_id
       t.string :title
       t.text :text
-      t.integer :comments_counter
-      t.integer :likes_counter
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
 
       t.timestamps
     end
-    add_foreign_key :posts, :users, column: :author_id, primary_key: 'id'
+    add_foreign_key :posts, :users, column: :author_id, primary_key: 'id', on_delete: :cascade
```

Comment:
- [ ] Great job here, the ERD shows `author_id` as a foreign key. A lot of students fail to understand how to work it out👏

Reply:
@elisha2kyakpo1 Thanks for your Review!